frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -352,6 +378,7 @@ function ReaderFooter:init()
         bookmark_count = true,
         time = true,
         page_progress = true,
+        page_remaining = true,
```

Comment:
We most probably want `page_remaining = false` - as having both page_progress and page_remaining enabled by default will confused people when they set "all at once".

(Also, it's the counterpart of `page_progress` so it's fine it's named `page_remaining` - but I would spell it more naturally as `pages_remaining` - plural. Dunno which is best.)

Reply:
A quick Google shows many ways to delete a commit - could you kindly suggest the "best" method to use here?

I named it `page_remaining` to match `page_progress` - should I change one or both of them?

Thanks!

Reply:
I guess you can't change `page_progress` as it's already used and saved in current users' settings.
I let @Frenzie tell us that `page_remaining` is just fine :) 

> could you kindly suggest the "best" method to use here?

Assuming you still have only these 2 commits:
```
git checkout add-pages-remaining
git reset --hard HEAD~1    # this resets 1 commit back, so removing your 2nd commit
git status   # check all looks fine
git log      # check all looks fine
git push -f origin add-pages-remaining
```

Reply:
> I named it page_remaining to match page_progress - should I change one or both of them?

@poire-z's point is that you say "page progress" and "pages remaining" (or "remaining pages") in natural language. ;-) I'd also prefer that s.

Much more important for variable naming though, is that we already have a `pages_left` which sounds completely synonymous. We should pick either left or remaining and add `_chapter` or `_in_chapter` to the one, `_book` or `_in_book` to the other, or both.

Reply:
> We should pick either left or remaining and add

But that's a bit tough for this PR, and a possible need for migration of previous settings names.

Reply:
True, but I was also talking about the menu text.

Also I think that implies that in principle the new setting should be called `pages_left_book` unless there's a sufficiently strong reason not to.